sql/executor.go

```diff
@@ -466,6 +481,20 @@ func pred(wr *PipeWriter, pr *extendedSelect, db *DB, cwd string, modelDir strin
 }
 
 func analyze(wr *PipeWriter, es *extendedSelect, db *DB, cwd string, modelDir string) error {
+	//pr, fts, e := loadModelMeta(es, db, cwd, modelDir)
```

Comment:
Remove the commented code?

Reply:
It is a sample code for how to use it.
Next PR will remove it.